Makefile

```diff
@@ -2,7 +2,7 @@
 
 check_prereqs:
 	bash -c '[[ -n $$VIRTUAL_ENV ]]'
-	bash -c '[[ $$(python3 --version) == *3.5.2* ]]'
+	bash -c '[[ $$(python3 --version) == *3* ]]'
```

Comment:
So we support any python3?

Reply:
Not necessarily, I removed it because the `3.5.2` was too restrictive.
This file is only used in circle ci which in turn uses an image with python v3.5.2, so it's not critical to have only 3.